### PR TITLE
All cooking effects + reflect + light projectile for qudobject.py

### PR DESCRIPTION
**Pull request by Wreckstation**
At least it should be.

### CODE REVIEW

File: qudobject.py [hunk 1]
```diff
@@ -645,6 +674,11 @@ def pvpowered(self):
         if self.vibro == 'yes' or self.part_Gaslight:
             return 'yes'
 
+    @property
+    def reflect(self):
+        """When preserved, how many preserves a preservable item produces."""
```
Comment (syntaxaire) [inline qudobject.py]: Comment needs update

File: config.yml [hunk 2]
```diff
@@ -28,13 +28,13 @@ Wiki:
     # Each wiki category is followed by a list of nodes in the inheritance tree whose child objects
     # should be in that category. Objects may also be explicitly specified.
     # Order matters since later, more specific entries can override earlier, more general ones.
```
Comment (syntaxaire) [inline config.yml]: The first things in the row are the wiki categories, the things between [] are what goes into those categories

File: qudobject.py [hunk 3]
```diff
@@ -478,7 +478,7 @@ def ego(self):
     @property
     def electric(self):
         """The elemental resistance/weakness the equipment or NPC has."""
-        return self.resistance('Electric')
+        return self.resistance('Elec')
```
Comment (syntaxaire) [inline qudobject.py]: Good catch

File: qudobject.py (hunk 1 above)
Comment (Wreckstation) [reply qudobject.py]: oops, fixed

**Pull request merged**